src/proxy/inbound_passthrough.rs

```diff
@@ -245,6 +245,6 @@ impl InboundPassthrough {
             }
             _signaled = close.signaled() => Err(Error::AuthorizationPolicyLateRejection)
         };
-        result_tracker.record(res);
+        result_tracker.record(res.map(|_| ()));
```

Comment:
What's with the empty closure?

Reply:
`res` it `Result<(u64,u64)>` but record takes `Result<()>`

Reply:
We could also drop all the send functions to stop tracking the bytes. Just moving this really.

Reply:
Oh, I see. We aren't using those retvals at all anymore - yeah the send funcs should probably not send them at all, it's a bit messy to throw them up to another layer which drops them in an empty closure.

Either add a TODO here explaining what's going on, or fix it, I am fine with either.

Reply:
done